javascript/javascript2/week1/spiritAnim.js

```diff
@@ -0,0 +1,28 @@
+const spiritAnimalName = [
+  "The Tiger",
+  "The Hawk",
+  "Rattle Snake",
+  "Cobra",
+  "The Eagle",
+  "Black Lion",
+  "Flying Mantis",
+  "Crouching Tiger",
+  "Black Scorpion",
+  "Quick Tortoise"
+];
+const buttonTag = document.getElementById("clickonMe");
+buttonTag.addEventListener("click", spiritAnimal);
+
+function spiritAnimal() {
+  const inputName = document.getElementById("name"); //u can also put .value here
```

Comment:
You are mixing DOM operations with logic. Your DOM code should be as simple as possible, and in this function you should just send the name as an argument.